### DISCUSSION TITLE
Error CefSharp.Core.Runtime.dll not found on 98.1.210

### DISCUSSION THREAD

**Discussion by araque-s** [cefsharp/CefSharp#4027] [General]
Hi!
- CefSharp.Wpf version 98.1.210
Windows 10 x64
Solution compiled on AnyCPU
Redist C++ 2015-2022 installed (x86-x64) - v14.31.31103

On this new version, when InitializeCefSharp, throws an error CefSharp.Core.Runtime.dll not found. With the same configuration and same code, on 98.1.190 it works.

If check the output folder contains required dlls and all the "extra" code to run with AnyCPU compilation is ok (as I say, on 98.1.190 works fine).

I'm not sure it's a personal bug or anyone found it. Thanks a lot!

**Comment by amaitland** [comment]
> On this new version, when InitializeCefSharp, throws an error CefSharp.Core.Runtime.dll not found. With the same configuration and same code, on 98.1.190 it works.

The only change between versions was a new chromium version. You can easily compare versions here on GitHub see https://github.com/cefsharp/CefSharp/compare/v98.1.190...v98.1.210

Start with a clean checkout.

**Comment by araque-s** [comment]
Yes, I make it work on a clean solution, but when I try to include in a previuos app, it doen't work with the last version, but it works on 98.1.190. I will continue the investigation. Thanks

**Reply by amaitland** [reply]
>  I make it work on a clean solution, but when I try to include in a previuos app, it doen't work with the last version

I meant perform a clean checkout of your application from your source repository. Alternatively delete all the bin/obj folders in your solution. 

Most often this is simply visual studio didn't copy all the new files and you have a mix of new and old in your bin folder.

**Reply by araque-s** [reply]
Great! You're right. I deleted bin/obj folders and reinstall the new version. It works fine!
Thanks for your support and, especially, for this tool. I find it very useful.
